cylc/flow/network/schema.py

```diff
@@ -753,8 +759,8 @@ class Meta:
             update relates to a workflow reload.
         '''),
     )
-    run_mode = String(
-        description="The scheduler's run-mode e.g. `live`.",
+    run_mode = WorkflowRunMode(
```

Comment:
The task run mode can be any of `skip, simulation, live, dummy` - does this allow that? It looks like it's only allowing the modes allowed in `WorkflowRunMode`.